libraries/resource_firewall_rule.rb

```diff
@@ -18,7 +18,12 @@ class Resource::FirewallRule < Chef::Resource::LWRPBase
                            !!(p.to_s =~ /(udp|tcp|icmp|icmpv6|ipv6-icmp|none)/ || (p.to_s =~ /^\d+$/ && p.between?(0, 142)))
                          end })
     attribute(:direction, kind_of: Symbol, equal_to: [:in, :out, :pre, :post], default: :in)
-    attribute(:logging, kind_of: Symbol, equal_to: [:connections, :packets])
+
+    if Chef::Platform.windows?
```

Comment:
Hi there -- if a downstream cookbook decides to use one of these attributes, to support windows and linux both, this will break it, right? I'd rather allow extra attribute values than prevent someone from supporting windows and linux when providing additional firewall rules in their community cookbook.